data/json/items/tool/med.json

```diff
@@ -410,5 +410,125 @@
     "flags": [ "BELT_CLIP", "SHEATH_KNIFE" ],
     "weapon_category": [ "SHIVS" ],
     "melee_damage": { "stab": 4 }
+  },
+  {
+    "id": "saline_bag_250",
+    "type": "GENERIC",
+    "category": "drugs",
+    "name": { "str": "small saline bag" },
+    "description": "A small, sealed plastic bag for liquids used in intravenous therapy, with rubber tube and cannula.  Holds 250 ml of completely sterile saline, therefore it is safe to put it inside your veins, if you know how.",
+    "looks_like": "bag_iv",
+    "material": [
+      { "type": "water", "portion": 991 },
+      { "type": "salt", "portion": 9 },
+      { "type": "plastic", "portion": 20 },
+      { "type": "rubber", "portion": 32 }
+    ],
+    "//": "250 g of solution, 5g of bag, 8 g of tube with cannula",
+    "weight": "263 g",
+    "volume": "295 ml",
+    "price": 999,
+    "price_postapoc": 25,
+    "symbol": ")",
+    "color": "light_gray",
+    "use_action": [
+      {
+        "type": "effect_on_conditions",
+        "menu_text": "Infuse saline",
+        "description": "You can use it to mitigate the blood loss consequences.",
```

Comment:
This line reads strangely.
Re-written it would read "You can use it to mitigate the consequences of blood loss."